src/index.js

```diff
@@ -82,7 +83,13 @@ const Image = require('./image.js');
  */
 class Compute extends common.Service {
   constructor(options = {}) {
-    options.apiEndpoint = options.apiEndpoint || 'compute.googleapis.com';
```

Comment:
Oh bother ...  guessing this isn't the only place I did that 😬   @stephenplusplus could I trouble you to do a broad search for this pattern in our libs (I use `@google/repo` to do this) and see where else I biffed it?